packages/transformer/src/test/standalone/IModelTransformerHub.test.ts

```diff
@@ -274,6 +274,151 @@ describe("IModelTransformerHub", () => {
     }
   });
 
+  it("should consolidate PhysicalModels", async () => {
+    const sourceIModelName: string = IModelTransformerTestUtils.generateUniqueName("ConsolidateModelsSource");
+    const sourceIModelId = await HubWrappers.recreateIModel({ accessToken, iTwinId, iModelName: sourceIModelName, noLocks: true });
+    assert.isTrue(Guid.isGuid(sourceIModelId));
+    const targetIModelName: string = IModelTransformerTestUtils.generateUniqueName("ConsolidateModelsTarget");
+    const targetIModelId = await HubWrappers.recreateIModel({ accessToken, iTwinId, iModelName: targetIModelName, noLocks: true });
+    assert.isTrue(Guid.isGuid(targetIModelId));
+
+    try {
+      // open/upgrade sourceDb
+      const sourceDb = await HubWrappers.downloadAndOpenBriefcase({ accessToken, iTwinId, iModelId: sourceIModelId });
+      const categoryId: Id64String = SpatialCategory.insert(sourceDb, IModel.dictionaryId, "SpatialCategory", { color: ColorDef.green.toJSON() });
+      const sourceModelIds: Id64Array = [];
+
+      const insertPhysicalObject = (physicalModelId: Id64String, modelIndex: number, originX: number, originY: number) => {
+        const physicalObjectProps1: PhysicalElementProps = {
```

Comment:
Instead of doing that with sqlite I noticed you can pass in an empty guid which causes the element to have an undefined  federationGuid after insert.
changed the code to insert every other element with an undefined fed guid

Reply:
that's perfect, I will also keep that in mind for the future and maybe update the other tests to do it that way eventually